dynamodb/dynamodb.go

```diff
@@ -184,6 +186,7 @@ var DefaultOptions = Options{
 	TableName:            "gokv",
 	ReadCapacityUnits:    5,
 	WriteCapacityUnits:   5,
+	CreateTable:          true
```

Comment:
trailing "," is missing here, see the Travis CI build: https://travis-ci.org/philippgille/gokv/jobs/638423386#L2320